fossid-webapp-client/src/main/kotlin/PostRequestBody.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.fossid
+
+class PostRequestBody(
+        val action: String,
+        val group: String,
+        apiKey: String,
+        user: String
+) {
+    val data: MutableMap<String, String> = mutableMapOf("username" to user, "key" to apiKey)
+}
```

Comment:
I have to admit I'm not a big fan of the strict / classic Java one-class-per-file approach. So how about putting all response body classes into a single `ResponseBody.kt` file?

Reply:
done